packages/core/src/components/Text.ts

```diff
@@ -44,9 +45,12 @@ export const TuiText = defineComponent({
     strikethrough: Boolean,
     inverse: Boolean,
     wrap: String as PropType<Styles['textWrap']>,
+    class: String,
   },
 
   setup(props, { slots }) {
+    props = transformClassIntoProps(props)
```

Comment:
Won't this break in terms of reactivity? 🤔 
It should be a computed instead (not indented)

```suggestion
   const propsWithClasses = computed(() => (props.class ?
     {
       ...props,
       ...transformClassToStyleProps(props.class),
     } : props))
```


I also think we should make the copy of `props` explicit here rather than putting it within the other function because it clearer what we are doing with it